src/window/User Guide.md

```diff
@@ -0,0 +1,132 @@
+% User Guide to Using Piston Windows
+
+# Guide to using Piston Windows
+
+## Getting Help
+
+*Notice! Piston is being actively developed. Please let us know if anything in this tutorial is outdated.*
+
+See the main [Piston Guide](../../GUIDE.md) for general help with Piston.
+
+## Overview
+
+The piston engine is designed to be an abstraction. It provides the same functionality through several different backends, each of which have their own quirks and extra features.
+
+Because of this abstraction, some of the details of the piston API may seem unnecessary or useless. The purpose of this guide is to give concrete guidelines on the creation and usage of piston windows.
+
+If you mostly know what you're doing, but need a quick refresher, see the quick reference at the bottom of this guide.
+
+# Windows
+
+## Choosing a Backend
+
+There are a few available backends for piston, and it's important to choose one for your project.
```

Comment:
If users only import this crate, there is no default behavior. The default glutin behavior exists in the piston_window crate, and so should be documented there.

If anything is a default in this crate, it'd be NoWindow. It's the only implementation in the whole crate.